webapp/boards/src/components/__snapshots__/rhsChannelBoards.test.tsx.snap

```diff
@@ -64,7 +64,7 @@ exports[`components/rhsChannelBoards renders the RHS for channel boards 1`] = `
           <div
             class="date"
           >
-            Last update at: July 08, 2022, 8:10 PM
+            Last update at: July 08, 2022 at 10:10 PM
```

Comment:
why was this snap updated? can you please revert

Reply:
done, sorry